buildkit/README.md

```diff
@@ -0,0 +1,44 @@
+# BuildKit
+
+This Task builds source into a container image using [Moby BuildKit](https://github.com/moby/buildkit).
+
+Rootless mode is used by default.
+
+## Parameters
```

Comment:
In the other catalog tasks, we have `## Inputs` and `### Parameters` and `### Resources`, and then `## Outputs` / `### Resources` -- can you do that here too?

Reply:
updated